src/Http/lib.mo

```diff
@@ -26,10 +26,18 @@ module {
     ********************/
 
     public func http_request_streaming_callback(token : Types.HttpStreamingCallbackToken) : Types.HttpStreamingCallbackResponse {
+      if (token.key == "placeholder") {
+        let placeholder = deps._Assets.getPlaceholder();
+        let res = _streamContent(token.key, token.index, placeholder.payload.data);
```

Comment:
`addPlaceholder` does not allow to chunk placeholders, so here we don't need to stream them i think? as we now only redirect to the assets stored in the asset canister, ideally we remove all this streaming/chunking stuff from the canister anyways. it adds unnecessary complexity to the http module.

Reply:
this would also mean we would need to upgrade our legacy canisters, but i think thats fine because we can just get rid of the placeholder entirely, as all those collections already launched and to not require the placeholder anymore. so in the long run we should have a migration for `clowns`